fsharp-backend/tests/testfiles/string.tests

```diff
@@ -62,6 +62,11 @@ String.toInt_v0 "-11111" = -11111
 String.toInt_v0 "+333" = 333
 String.toInt_v0 "-11111" = -11111
 String.toInt_v1 "1" = 1
+String.toFloat_v0 "1.5" = 1.5
```

Comment:
Can you add a test for a large float (larger than would fit in 32 bits)? The code looks right, I just like to handle edge cases.